javaparser-core/src/main/java/com/github/javaparser/ast/modules/ModuleDeclaration.java

```diff
@@ -197,4 +202,16 @@ public boolean replace(Node node, Node replacementNode) {
         }
         return super.replace(node, replacementNode);
     }
+
+    /**
+     * Add a directive to the module, like "exports R.S to T1.U1, T2.U2;"
+     */
+    public ModuleDeclaration addDirective(String directive) {
+        return addDirective(JavaParser.parseModuleDirective(directive));
+    }
+
+    public ModuleDeclaration addDirective(ModuleStmt directive) {
```

Comment:
also naughty?

Reply:
Indeed!

Reply:
Ok, just thought because its sibling got docs, but that is probably because it is a string that has to be properly formatted. :-)